### PR TITLE
zookeeper 3.8.0

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

continuing #96415

### CODE REVIEW

File: Formula/zookeeper.rb [hunk 1]
```diff
@@ -34,14 +33,36 @@ def default_zk_env
     EOS
   end
 
-  def default_log4j_properties
+  def default_logback_xml
```
Comment (SMillerDev) [inline Formula/zookeeper.rb:36]: Is there a resource we can download this from?

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/zookeeper.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/zookeeper.rb:36]: it got customized from [this file](https://github.com/apache/zookeeper/blob/master/conf/logback.xml)

Comment (SMillerDev) [reply Formula/zookeeper.rb:36]: I really don't think we should be making our own config. Let's just download that file instead.

Comment (chenrui333) [reply Formula/zookeeper.rb:36]: Sounds good, let me pull the sample one

**Pull request closed without merging**